### ISSUE TITLE
Github Action to comment on PRs during develop freeze

### ISSUE THREAD

**Issue by moniika** [google/blockly#4971]
**Is your feature request related to a problem? Please describe.**
During release week we freeze develop and do not merge in changes unless they are considered high priority fixes that need to get in for the release. While develop is frozen, PR reviews are generally postponed until after release and sometimes it is forgotten that changes shouldn't be merged in. This can also be an unpleasant experience for contributors waiting on a review and not hearing back,

**Describe the solution you'd like**
An automatic comment during release week to remind contributors that develop is frozen and that their PR probably wont be merged in until after release and that reviews of their PR may be delayed.

**Describe alternatives you've considered**
Add a label to designate that this request will be postponed till after release (or maybe to say that it is targeting the next release milestone).

**Additional context**
Similar implementations:
https://github.com/actions/first-interaction/blob/main/src/main.ts

**Comment by cpcallen** [comment]
Tangential question: is there a reason you freeze `develop`, rather than cutting a release branch and letting `develop` continue as usual?

I asked @NeilFraser, and he said that (so far as remembered) the team had previously used release branches to avoid a freeze, but couldn't recall why you no longer did that.

**Comment by alschmiedt** [comment]
If I am remembering correctly the main reason was that we usually had a few bug fixes that end up needing to go in after we cut the release candidate. Every time we had a bug that needed to go in we would have to merge to develop and then cherry pick into the release candidate. There were other reasons that I could probably dig up if you are interested, but in general we tried it for a release and decided that it made the process a lot less convoluted.

**Comment by rachel-fenichel** [comment]
At the time that we decided to just freeze develop all of our devs were directly involved in the release, and so holding PRs that weren't release related only impacted 1 or 2 PRs a quarter from external developers. 

With more devs we may want to reconsider.
